LAB6.ipynb

```diff
@@ -0,0 +1,747 @@
UnWHXKYyIRyLi4fL264h4IiIebkc4SZLqJCI+DtwKPALMzswPWBBK6lSuV6iGVs4UHtp8PyJOB44fs0SSJNXXX1Bcf/8+4L0R0WgPiolmnlZVMK8plDSU1eu2VB1BFRr2TGF/mXk1+7BGodQV+npg5bxisWBJXSczD8jMCZl5aGY+rel2aJUFYZntmsxcPHHixCpjSJI60LBnCiPizKa7BwBzKdYZlNTfhquKgnDSHJj2uqrTSJK6UV/Pnv2RJA2jldlHFzRt76RYdH7hmKSRxoNJc+DEVUxbci1wbdVpJEndxi8oJY1QK9cUnteOIJLqozF99UUfu57erfPZsPTUqiNJkpqVX1BKUisGLQoj4v1DPC8zs7IFeCVVZ4/pqylmLpMkSVJ9DTXRzC8HuAGcD7x7jHNJ6lBOXy0Nr99yTo3bfRHxTxExo6JMCyKiZ8eOHVW8vaQOd8Ihaznn8OurjqGKDFoUZuYnGjegB5gAnAcsAyrp0CRJqolPUqwCPQWYCvwl8AWKPvTyKgI5+6ikwTRGAS08bFWlOVSdIZekiIjDI+KvgDsohpoel5nvzszNbUknSVI9zc/Mz2fmI5n5cGb2AKdk5leBSVWHk6RmvVvns/rRY6uOoQoNWhRGxMeBW4FHgNmZ+YHM3Na2ZJIk1ddvI+LsiDigvJ3d9JjLOkmSOspQZwr/AngW8D7g/qZrIh6JiIfbE0+SpFp6PfBGYHN5eyPwhoiYAFxYZTBJkvobdPbRzBxyaKkkSRpYZq5jz3V+m/2wnVkkSRqOhZ8kSaMsIqaWM41ujogHIuKbETG16lySJA1k2MXrJUnSiP09cBVwVnn/DWXbSVUFiogFwIKZM2dWFUFjra8HNlwF29YUi9cD05ZcW3Eo1VHjc7Nh6akVJ1G7eKZQGg19PbByXtERSxJMzsy/z8yd5e0KYHKVgVySogs0F4TTXld1GtXQrAnrWTZjiesVdiHPFEqjoakjvujHs+ld6TezUpd7KCLeAPSW988BtlSYR91i0hw4cVXVKVRDjbUKZ01YDxTLVKh7eKZQGi1lR9xNv0Qb3yjS11N1FKnTvBk4G/g3YBPwGuC8ShNJ0hB6t85n0bql3PXY9KqjqAIWhZL2yfLt87jrsenMmrCe1d+91OtWpCaZ+fPMPC0zJ2fmEZl5OnBm1bkkSRqIRaGkfeI3itKIvavqAJIkDWTMisKIuLycinttU9vhEXFDRNxb/jmpbI+IuCQi+iLijog4ruk555b73xsR545VXkmSxlhUHUCSpIGM5ZnCK4D+F1ctAW7MzGOAG8v7ACcDx5S3xcBnoSgigYuBFwPHAxc3CklJkmomqw4gSdJAxqwozMzvA1v7NS8Eriy3rwROb2r/chZWA4dFxJHAq4AbMnNrZm4DbmDvQlOSpI4QEY9ExMMD3B4BnlV1PkmSBtLuJSmemZmbADJzU0QcUbZPAe5r2m9j2TZYuyRJHSczD606gyRJI9UpE80MdJ1FDtG+9wtELI6I2yLitgcffHBUw0mSJEndwiWnuk+7i8IHymGhlH9uLts3Akc17TcVuH+I9r1kZk9mzs3MuZMnTx714JIk1VlELIiInh07dlQdRaOtrwdWzoNta6pOonFgqCWnpi25dsj7qq92F4UrgMYMoucCy5va31TOQnoCsKMcZvpt4JURMamcYOaVZZskSRqBzLwmMxdPnDix6igabRuuKgrCSXNg2uuqTqOac8mp7jRm1xRGRC8wD3hGRGykmEV0KfC1iDgf+DlwVrn7dcApQB/wK+A8gMzcGhEfBm4t9/tQZvafvEaqTl/Prs549bajWNSl35btHmbyC5i5uOo4ktR9Js2BE1dVnUJSTY1ZUZiZ5wzy0CsG2DeBtw3yOpcDl49iNGn0NH07u3z97KrTVGL59nlAURiy4SqLQkmSpJrplIlmpPoqv53t3dqdq6U4zESSJKneLAolSZIkqYtZFEqSJElSF7MolDRqVq/b4tTUkiRJNWNRKEmSJGkvLmLfPSwKJUmSJO2heRF7NlxVdRyNMYtCSZIkSXtwdvHuYlEo7Sevo9vNYSaSJEn1Y1Eo7au+Hth8U9UpOobDTCSpzfp6YOU82Lam6iSSas6iUNpXZeGzfPu8anN0iOZhJp49laQ22HBVURBOmgPTXld1Gkk1dlDVAaRaO+Kl9N4xv+oUkrpURDwV+AzwG2BVZn6l4khqt0lz4MRVVaeQVHOeKZQkqYNExOURsTki1vZrnx8R90REX0QsKZvPBL6RmRcAp7U9rCRpXLAolCSps1wB7DEEISIOBC4DTgZmAedExCxgKnBfudsTbcwoSRpHLAolSeogmfl9YGu/5uOBvsxcl5m/AZYBC4GNFIUh2KdLkvaR1xRKktT5prD7jCAUxeCLgUuAT0fEqcA1gz05IhYDiwGOPvroMYypsdaYxGvD0lOHfFwaTQ9vuo27emZzzuHz6N06f9DP2XCfT3Uui0JJkjpfDNCWmflL4LzhnpyZPUAPwNy5c3OUs0kaxxqzrM+asB4oZhvX+ONQE2k/rF63peoIHclF7KVRtxE4qun+VOD+kbxARCyIiJ4dO3aMajBJ41vzklMavywKpZFyseAhuYi9NCZuBY6JiOkR8SRgEbBiJC+Qmddk5uKJEyeOSUBJUn1ZFEoj1bRYsAvX781F7KX9ExG9wP8FnhsRGyPi/MzcCVwIfBu4G/haZt5ZZU5J0vjhNYXSvigXC+5dacEjaXRl5jmDtF8HXLevrxsRC4AFM2fO3NeXkCSNU54plCSpCzh8VJI0GItCSZKkGjnn8Ou9tl3SqHL4qKQx57pFkjR6Fh62CrbdV1zKMO11VceRNA54plDSmGksTXHO4ddXHUXqei5JMc6U17Yzc3HVSdRF7NfHr0qKwojYEBE/jYg1EXFb2XZ4RNwQEfeWf04q2yMiLomIvoi4IyKOqyKzpJFpXppi4WGrqo4jdT2vKZS0P+zXx7cqzxS+LDPnZObc8v4S4Mb
```

Comment:
The first sentence needs to be a bit more clear when you talk about signal strength. Instead of saying "signal strength" say "injected signal strength." Rewording the last part of this sentence so that it matched the format of the first part might help with this as well.